demoScripts/quickstart-nodejs.md

```diff
@@ -0,0 +1,529 @@
+        <p>Welcome to Azure</p>
+      </body>
+    </html>
+    ```
+
+::: zone-end
+
+## Stream Logs
+
+:::zone target="docs" pivot="development-environment-vscode"
+
+You can stream log output (calls to `console.log()`) from the Azure app directly in the Visual Studio Code output window.
+
+1. In the **App Service** explorer, right-click the app node and select **Start Streaming Logs**.
+
+    ![Start Streaming Logs](./media/quickstart-nodejs/view-logs.png)
+
+1. If asked to restart the app, select **Yes**. Once the app is restarted, the Visual Studio Code output window opens with a connection to the log stream.
+
+1. After a few seconds, the output window shows a message indicating that you're connected to the log-streaming service. You can generate more output activity by refreshing the page in the browser.
+
+    <pre>
+    Connecting to log stream...
+    2020-03-04T19:29:44  Welcome, you are now connected to log-streaming service. The default timeout is 2 hours.
+    Change the timeout with the App Setting SCM_LOGSTREAM_TIMEOUT (in seconds).
+    </pre>
+
+> [!div class="nextstepaction"]
+> [I ran into an issue](https://www.research.net/r/PWZWZ52?tutorial=node-deployment-azure-app-service&step=tailing-logs)
+
+::: zone-end
+
+:::zone target="docs" pivot="development-environment-cli"
+
+You can access the console logs generated from inside the app and the container in which it runs. Logs include any output generated by calls to `console.log()`.
+
+To stream logs, run the [az webapp log tail](/cli/azure/webapp/log#az-webapp-log-tail) command:
+* the following command will stream logs for 30 seconds before exiting.
+```azurecli
+timeout 30s az webapp log tail
+```
+
+The command uses the resource group name cached in the *.azure/config* file.
+
+You can also include the `--logs` parameter with then [az webapp up](/cli/azure/webapp#az-webapp-up) command to automatically open the log stream on deployment.
+
+Refresh the app in the browser to generate console logs, which include messages describing HTTP requests to the app. If no output appears immediately, try again in 30 seconds.
+
+To stop log streaming at any time, press **Ctrl**+**C** in the terminal.
+
+::: zone-end
+
+:::zone target="docs" pivot="development-environment-azure-portal"
+
+You can access the console logs generated from inside the app and the container in which it runs. You can stream log output (calls to `console.log()`) from the Node.js app directly in the Azure portal.
+
+1. In the same **App Service** page for your app, use the left menu to scroll to the *Monitoring* section and select **Log stream**.
+
+    :::image type="content" source="./media/quickstart-nodejs/log-stream.png" alt-text="Screenshot of Log stream in Azure App service.":::
+
+1. After a few seconds, the output window shows a message indicating that you're connected to the log-streaming service. You can generate more output activity by refreshing the page in the browser.
+
+    <pre>
```

Comment:
This looks like the results block for above. This zone thing is crazy, it makes it super hard to read the md. I feel dirty even looking at it.